### ISSUE TITLE
Most commands fail when special emoji isn't set

### ISSUE THREAD

**Issue by laggron42** [Ballsdex-Team/BallsDex-DiscordBot#77]
[This `try/except` block](https://github.com/Ballsdex-Team/BallsDex-DiscordBot/blob/13e0ca1a4b0de3996b72545556c3a2e37a2296f4/ballsdex/core/models.py#L230-L236) needs to catch `TypeError` in addition to `ValueError`.

Error below when using `/balls list`

```
2023-04-20 02:53:09 ERROR    ballsdex.core.bot Error in slash command list
Traceback (most recent call last):
  File "/usr/local/lib/python3.11/site-packages/discord/app_commands/commands.py", line 841, in _do_call
    return await self._callback(self.binding, interaction, **params)  # type: ignore
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/code/ballsdex/packages/players/cog.py", line 156, in list
    await paginator.start()
  File "/code/ballsdex/core/utils/paginator.py", line 192, in start
    kwargs = await self._get_kwargs_from_page(page)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/code/ballsdex/core/utils/paginator.py", line 79, in _get_kwargs_from_page
    value = await discord.utils.maybe_coroutine(self.source.format_page, self, page)
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/discord/utils.py", line 688, in maybe_coroutine
    return await value
           ^^^^^^^^^^^
  File "/code/ballsdex/packages/players/countryballs_paginator.py", line 20, in format_page
    menu.set_options(balls)
  File "/code/ballsdex/packages/players/countryballs_paginator.py", line 37, in set_options
    special = ball.special_emoji(self.bot, False)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/code/ballsdex/core/models.py", line 231, in special_emoji
    emoji_id = int(self.specialcard.emoji)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^
TypeError: int() argument must be a string, a bytes-like object or a real number, not 'NoneType'
```

**Comment by laggron42** [comment]
Fixed by #71

**Comment by laggron42** [comment]
Oops
